spinc/cmd/status.go

```diff
@@ -75,3 +77,7 @@ func (c *Status) Run() error {
 func (c *Status) Cmd() string {
 	return "status " + c.reqId
 }
+
+func (c *Status) Help() string {
+	return "'spinc status <request ID>' prints the status of the request.\n"
```

Comment:
I'd mention -v for verbose output (i.e. running job status).

Reply:
Let's hold off on that because I'm going to rework the spinc output and remove `-v`.